client/store/ui/messageCenter/slice.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice, Reducer } from '@reduxjs/toolkit'
+import { MessageCenterState } from './stateType'
+
+const initialState: MessageCenterState = {
+  topics: Array<MessageTopic>(),
+}
+
+export const messageCenterSlice = createSlice({
+  name: 'messageCenter',
+  initialState,
+  reducers: {
+    reset: () => initialState,
+    open: (state, action) => {
+      if (state.topics.length < 2) state.topics.push(action.payload)
```

Comment:
it should always be added.
the first entered should be removed !

Reply:
if (state.topics.length === 2) state.topics.shift()
      state.topics.push(action.payload)